contrib/docker/docker-entrypoint.sh

```diff
@@ -134,7 +139,9 @@ elif [ "$1" = 'zammad-railsserver' ]; then
   echo "starting railsserver... with WEB_CONCURRENCY=${ZAMMAD_WEB_CONCURRENCY}"
 
   #shellcheck disable=SC2101
-  exec bundle exec puma -b tcp://[::]:"${ZAMMAD_RAILSSERVER_PORT}" -w "${ZAMMAD_WEB_CONCURRENCY}" -e "${RAILS_ENV}"
+  if [ "$ZAMMAD_DISABLE_IPV6"==1 ]; then
+    exec bundle exec puma -b tcp://0.0.0.0:"${ZAMMAD_RAILSSERVER_PORT}" -w "${ZAMMAD_WEB_CONCURRENCY}" -e "${RAILS_ENV}"
+  fi
```

Comment:
Here it seems that an `else` is missing, the default IPv6 case. To avoid redundancy, we could consider declaring a PUMA_BIND_ADDRESS with the default IPv6 value, and changing it in the `if` statement to the IPv4 value. Then only one puma call can be made afterwards.

It would also be good to add the variable to the top of the file where all variables are initialized (default to empty).